### PR TITLE
added paper for later use

### PR DESCRIPTION
**Pull request by wdecker**

### CODE REVIEW

File: docs/oscar_references.bib [hunk 1]
```diff
@@ -63,6 +63,38 @@ @InProceedings{BDLP19
...
+  journal       = {Journal of Symbolic Computation},
+  volume        = {51},
+  note          = {Effective Methods in Algebraic Geometry},
+  pages         = {99-114},
+  year          = {2013},
+  doi           = {https://doi.org/10.1016/j.jsc.2012.07.002},
+  url           = {https://www.sciencedirect.com/science/article/pii/S0747717112001186},
+  abstract      = {Given a reduced affine algebra A over a perfect field K,
+                  we present parallel algorithms to compute the normalization
+                  A¯ of A. Our starting point is the algorithm of Greuel et
```
Comment (fingolfin) [inline docs/oscar_references.bib]: What is "A¯ of A" ? Maybe just omit the whole abstract?

File: docs/oscar_references.bib [hunk 2]
```diff
@@ -63,6 +63,38 @@ @InProceedings{BDLP19
...
+@Article{BDLPSS13,
+  author        = {Janko Böhm and Wolfram Decker and Santiago Laplagne and
+                  Gerhard Pfister and Andreas Steenpaß and Stefan Steidel},
+  title         = {Parallel algorithms for normalization},
+  journal       = {Journal of Symbolic Computation},
+  volume        = {51},
+  note          = {Effective Methods in Algebraic Geometry},
+  pages         = {99-114},
+  year          = {2013},
+  doi           = {https://doi.org/10.1016/j.jsc.2012.07.002},
```
Comment (fingolfin) [inline docs/oscar_references.bib]:
```suggestion
  doi           = {10.1016/j.jsc.2012.07.002},
```

File: docs/oscar_references.bib (hunk 1 above)
Comment (wdecker) [reply docs/oscar_references.bib]: As the text says, A¯ is the normalization of A. But please feel free to remove the abstract.

File: docs/oscar_references.bib (hunk 2 above)
Comment (wdecker) [reply docs/oscar_references.bib]: As above, please change.

File: docs/oscar_references.bib [hunk 3]
```diff
@@ -63,6 +63,38 @@ @InProceedings{BDLP19
...
+                  where K=Q is the field of rationals, we propose modular
+                  versions of the global and local-to-global algorithms. We
+                  have implemented our algorithms in the computer algebra
+                  system Singular and compare their performance with that of
+                  the algorithm of Greuel et al. (2010). In the case where
+                  K=Q, we also discuss the use of modular computations of
+                  Gröbner bases, radicals, and primary decompositions. We
+                  point out that in most examples, the new algorithms
+                  outperform the algorithm of Greuel et al. (2010) by far,
+                  even if we do not run them in parallel.}
```
Comment (fingolfin) [inline docs/oscar_references.bib]:
```suggestion
  url           = {https://www.sciencedirect.com/science/article/pii/S0747717112001186}
```

**Review by fingolfin — APPROVED** [review]

**Pull request merged**